static/scss/vendors/_bootstrap-custom.scss

```diff
@@ -0,0 +1,68 @@
+/*!
+ * Bootstrap v3.3.6 (http://getbootstrap.com)
+ * Copyright 2011-2015 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
```

Comment:
I'd like to avoid switching to bootstrap, and not have to add it to the dependencies - we would be adding technical debt given that the rest of the interface is using Foundation.